improving.html

```diff
@@ -111,16 +111,16 @@ <h2 id="understand">What is Accessibility?</h2>
 	<p>If there is a need, more in-depth introductions and background is provided in <a href="https://www.w3.org/WAI/intro/people-use-web/">How People with Disabilities Use the Web</a>. Specifically the <a href="https://www.w3.org/WAI/intro/people-use-web/principles">Accessibility Principles</a> section of the resource introduces the accessibility requirements and guidelines.</p>
 
 	<h2 id="explore">What are the Issues?</h2>
-	<p>If you already know the specific issues you need to address, you can skip this section. If you are starting to address accessibility without background, the following manual helps you get a rough idea of the current situation on your website:</p>
+	<p>If you already know the specific issues you need to address, you can skip this section. If you are starting to address accessibility without background, the following manual helps you get a rough idea of the current situation on your web project:</p>
 	<ul>
 		<li><a href="https://www.w3.org/WAI/eval/preliminary.html">Easy Checks &mdash; A First Review of Web Accessibility</a> &mdash; Simple checks that can be carried out by anyone, regardless of technical skills and accessibility knowledge.</li>
 	</ul>
 	<p>Some web accessibility evaluation tools may help you further explore the accessibility on your website:</p>
 	<ul>
 		<li><a href="https://www.w3.org/WAI/ER/tools/">List of Web Accessibility Evaluation Tools</a> &mdash; Customizable list that allows you to search for different types of tools.</li>
 	</ul>
-	<p>Note: some web accessibility evaluation tools may be complex and confusing to use for people new to accessibility. They typically only check a subset of issues, and may be prone to misleading results.</p>
-	<p>To get a complete picture of the situation, a comprehensive evaluation may be needed. Such thorough evaluations require accessibility expertise. The following resources help you evaluate your website:</p>
+	<p>Note: Some web accessibility evaluation tools may be complex and confusing to use for people new to accessibility. They typically only check a subset of issues, and their results may need to be interpreted to avoid being misleading.</p>
```

Comment:
Another issue with the second sentence... The word "They" at the beginning of the sentence is referring to the "tools," but some could interpret that it is referring to "people new to accessibility."

Maybe try: "Note: Some web accessibility evaluation tools may be complex and confusing to use for people new to accessibility. Some tools may only check a subset of issues, and they may need expertise to interpret the results.